### DISCUSSION TITLE
Custom Dashboard Help?

### DISCUSSION THREAD

**Discussion by BlueFalconHD** [LunarVim/LunarVim#3569] [Q&A]
In this configuration file: https://github.com/AmitGolden/dotfiles/blob/main/.config/nvim/lua/user/alpha.lua, I am trying to make it work in LunarVim. I have it in my configuration, but it doesn't seem to work. Can anyone help me port this to LunarVim?

**Comment by mamachanko** [comment]
@BlueFalconHD I think alpha's configuration needs to go onto `lvim.builtin.alpha` as per [docs on core plugins](https://www.lunarvim.org/docs/plugins#core-plugins).

**Reply by BlueFalconHD** [reply]
Yep. I am aware of this. I just cannot figure out how to transfer it. The structure of the Alpha builtin is way different that normal Alpha.
